runtime/gateway.go

```diff
@@ -413,6 +424,25 @@ func (gateway *Gateway) setupLogger(config *StaticConfig) error {
 	return nil
 }
 
+func (gateway *Gateway) setupTracer(config *StaticConfig) error {
+	opts := []jaegerConfig.Option{
+		// TChannel logger implements jaeger logger interface
+		jaegerConfig.Logger(NewTChannelLogger(gateway.Logger)),
+		jaegerConfig.Metrics(NewJaegerMetricsFactory(gateway.MetricScope)),
+	}
+	// TODO: using default tracing configurations here, we may want to set it from static config
+	jc := jaegerConfig.Configuration{}
+	serviceName := config.MustGetString("serviceName")
+	closer, err := jc.InitGlobalTracer(serviceName, opts...)
```

Comment:
We can, but this is actually the [recommended](https://godoc.org/github.com/uber/jaeger-client-go/config#example-Configuration-InitGlobalTracer-Production) way to initialize the tracer per jaeger. The benefit is that the tracer ownership is managed by the opentracing package, so one can start a span without the need to figure out where to get the tracer.